CIScripts/Test/Tests/vRouterAgentService.Tests.ps1

```diff
@@ -0,0 +1,102 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestbedAddr
+)
+
+. $PSScriptRoot\..\Utils\CommonTestCode.ps1
+. $PSScriptRoot\..\TestConfigurationUtils.ps1
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+. $PSScriptRoot\..\..\Common\VMUtils.ps1
+. $PSScriptRoot\..\PesterHelpers\PesterHelpers.ps1
+
+. $PSScriptRoot\..\GetTestConfigurationCodiLegacy.ps1
```

Comment:
We should either use the file from environment (at least temporarily) or merge those files into one.

Reply:
I don't know what you mean. I am not going to expand this PR further to clean up the config file mess if that's what you guys want :smile: